app/metal-controller-manager/api/v1alpha1/environment_types.go

```diff
@@ -76,7 +76,7 @@ type EnvironmentList struct {
 func EnvironmentDefaultSpec(talosRelease, apiEndpoint string, apiPort uint16) *EnvironmentSpec {
 	args := make([]string, 0, len(kernel.DefaultArgs)+6)
 	args = append(args, kernel.DefaultArgs...)
-	args = append(args, "console=tty0", "console=ttyS1,115200n8", "earlyprintk=ttyS1,115200n8")
```

Comment:
this fixes default environment console args. we can't guess whether it's serial port zero or one, but providing zero makes more sense, as it matches Talos defaults on platform `metal` and also fixes Sfyra console output